File: src/components/EmptyState.js
import React from 'react';
import {StyleSheet, Text, View} from 'react-native';
import colors from '../constants/colors';

export default function EmptyState() {
  return (
    <View style={styles.emptyState}>
      <Text style={styles.emptyText}>No tasks scheduled for today.</Text>
      <Text style={styles.emptySubtext}>Plan your day and stay productive.</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  emptyState: {
    backgroundColor: colors.surface,
    borderRadius: 12,
    padding: 18,
    marginBottom: 12,
  },
  emptyText: {
    color: colors.text,
    fontSize: 17,
    fontWeight: '600',
    marginBottom: 6,
  },
  emptySubtext: {
    color: colors.muted,
    fontSize: 14,
  },
});

File: lib/widgets/empty_state.dart
import 'package:flutter/material.dart';

class EmptyState extends StatelessWidget {
  const EmptyState({super.key});

  @override
  Widget build(BuildContext context) {
    return Center(
      child: Padding(
        padding: const EdgeInsets.all(24.0),
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: const [
            const Icon(Icons.event_available, size: 64, color: Colors.grey),
            const SizedBox(height: 16),
            const Text('No tasks scheduled for today.',
                style: TextStyle(fontSize: 16, fontWeight: FontWeight.w600)),
            const SizedBox(height: 8),
            const Text('Plan your day and stay productive.',
                style: TextStyle(color: Colors.grey)),
          ],
        ),
      ),
    );
  }
}